src/components/application_manager/include/application_manager/resumption/resumption_data_processor.h

```diff
@@ -125,6 +125,17 @@ class ResumptionDataProcessor : public app_mngr::event_engine::EventObserver {
                        const hmi_apis::FunctionID::eType);
 
  private:
+  utils::Optional<uint32_t> GetAppIdByRequestId(
+      const hmi_apis::FunctionID::eType function_id, const int32_t corr_id);
+
+  std::vector<ResumptionRequest>::iterator GetRequest(
```

Comment:
Please do not return iterators, it not good practice

Reply:
Already fixed in 
b08ff60